streams/test-utils/src/main/java/org/apache/kafka/streams/TopologyTestDriver.java

```diff
@@ -213,7 +214,9 @@
     ProcessorTopology processorTopology;
     ProcessorTopology globalTopology;
 
+    private final MockConsumer<byte[], byte[]> consumer;
     private final MockProducer<byte[], byte[]> producer;
+    private final StreamsProducer streamsProducer;
```

Comment:
We need to commit explicitly in TTD now to mimic the TaskManger. Hence, we need access to the `consumer` and `streamsProducer`

Reply:
Makes sense to me.